api/proto/info/info.proto

```diff
@@ -918,3 +920,25 @@ message ZInfoAppInstMetaData {
   AppInstMetaDataType type = 2;
   bytes data = 3;
 }
+
+// EvAccessType - edge-view access type
+enum EvAccessType {
+  EV_ACCESS_TYPE_UNSPECIFIED = 0;        // not used, invalid access type
+  EV_ACCESS_TYPE_DEV         = 1;        // access to device side
+  EV_ACCESS_TYPE_APP         = 2;        // access to application side
+  EV_ACCESS_TYPE_EXT         = 3;        // access to external of device
+}
+
+// ZInfoEdgeview - edge-view operation status/event
+message ZInfoEdgeview {
+  // started timestamp
+  google.protobuf.Timestamp started_time = 1;
+  // total number of cmds edge-view has processed
+  uint32 count = 2;
+  // edge-view option such as tcp/locolhost:22, tcp/localhost:5902; shell/reboot, etc
+  // when the cmd operation is finished, cmdOption string will be reset to empty,
+  // unless there is another cmdOption within a specified time frame
+  string cmd_option = 3;
```

Comment:
Is the intent that this cmd_option in the info messages can be used to see all of the commands which are excuted?
That wouldn't work since we assume that if we have a failure to send an info message then when we retry sending that info message we will include the most recent information.
This if cmd_option was xyzzy and that failire to send, then the operation finished and we attempt send an info with cmd_option being empty, the controller would never see that xyzzy was performed.

Reply:
Right. Although the cmds are also saved into device logs; what about we adding in /persist/ a edge-view directory, and have a circular buffer file to record all the important commands from edge-view, and keep them for weeks?

Reply:
If the logs already have them we can find them. Additional tools can make them easier to find from the logs. So we don't need any additional mechanism IMHO.

Reply:
ok.

Reply:
But we probably want to keep the cmd_option set until it has been successfully sent to the controller.
One way to do this is to add a "completed" boolean so that normally we would first send 
  cmd_option=xyzzy; completed=false
and a bit later
  cmd_option=xyzzy; completed= true

However, should the first fail the controller will still see the second message as long as we add code to not remove a cmd options to be sent until we have successfully sent it with completed=true

Does such an approach make sense?

Reply:
Sure. Although the previous intention was to send one notification even on every 'important cmd' (not like show route, etc), so when it fails, there might be several others. So we need to keep a slice on all the outstanding 'important cmds' there and mark them altogether.